azure-pipelines.yml

```diff
@@ -50,6 +50,7 @@ pr:
     - master
     - release/3.0
     - release/3.0-preview9 
+    - internal/release/3.*
```

Comment:
Why do you need it on the master branch?
And why not have `- release/*`?

Reply:
In master, it should have been `release/*` and not `release/3.*`. I'll fix this up again when I merge https://github.com/dotnet/wpf/pull/2205. 

`internal/*` branches exist for non-public development, for e.g., security fixes. They are not intended to be part of the GitHub repo - they exist to be used from the the internal AzDo mirror repo.

Reply:
> `internal/*` branches exist for non-public development, for e.g., security fixes.

My question was more about why you added `- internal/release/3.*` to the master branch. It would make sense to have it as `- internal/release/*`